data/hai/unfettered 0 prologue.txt

```diff
@@ -515,3 +517,1010 @@ mission "Unfettered Jump Drive Trading"
+				decline
+			label shoot
+			`	You get your weapon out and shoot the monster as it attempts to rush out of its aquarium. You are not sure if your shot over the aquarium hit it, because of the erupting water that came out your way as the beast emerged, but at least you are still alive!`
+				goto reward
+
+event "unfettered: JD raid plan ready"
+
+
+
+event "cerberus revealed"
+	government "Cerberus"
+		remove "display name"
+
+mission "<uhai> 3: Recover Jump Drives"
+	name `Recover the jump drives`
+	description `Go to the pirate world of <stopovers> and recover 4 jump drives from a vault.`
+	source "Darkcloak"
+	stopover "Freedom"
+	clearance
+	passengers 1
+	to offer
+		has "event: unfettered: JD raid plan ready"
+	on offer
+		conversation
+			`As you pass what looks like a bar in the spaceport, you're greeted by the same Hai captain that challenged you earlier. "So we meet again! Arr, if ye be wanting to scuttle me clipper again, it'll take a lot more salt out of ye! Well... once I have her fixed and back, anyway. But that not be why I be here, today," he says, apparently happy to see you. "I have a job for ye!"`
+			choice
+				`	"What's the job about?"`
+					goto job
+				`	"Sure, I would do anything for my favorite squirrel!"`
+					goto insult
+				`	"Just admit it. You're scared to fight me."`
+			`	"Nice try, but ye will not trick this old cap'n! Worry not, soon I shall have a grander vessel, and when I do, 'twill be ye that walks the plank in our next battle! Most of the time, having the bigger guns on yer man-o-war is all the strategy and diplomacy ye need. In this case, however, I be lacking cannons, so the best choice is for me to fold. For now, at least."`
+			choice
+				`	"What can I do for my favorite squirrel?"`
+					goto insult
+				`	"How soon will you have that new ship?"`
+				`	"Well, just do it with that bigger ship of yours, then. I'm not interested."`
+					decline
+			`	"Once this job be done, hopefully! It should pay handsomely - 'buying a brand new ship' kind of handsome." Judging from your last encounter, this will probably be a very dangerous mission, but it may well be worth it.`
+			choice
+				`	"What kind of job is it?"`
+					goto job
+			label insult
+			`	He does not seem to react to the insult. He did call you monkey a few times before, so it's only fair for you to return the courtesy.`
+			label job
+			`	"We be going to collect a few jump drives! Normally ye would not be able to get such a job, since ye have not earned much among us still, but I have the connection to get it for ye. Ye see, officially this be me mission, and a very lucrative one, at that! On top of that, should we succeed, next time ye may be entrusted with such missions on your own, but don't ye be expecting near the same booty for yerself."`
+			choice
+				`	"Okay... That just seems too good to be true. What's the catch?"`
```

Comment:
```suggestion
				`	"Okay... that just seems too good to be true. What's the catch?"`
					`	"And why wouldn't I want to?"`
```
To give the player more options.

Reply:
I'm not sure about that option in this context but I could just be a bit confused

Reply:
Some option should be given though. Going to the same thing, of course.